### PR TITLE
[6.x] Core/ObjectMgr: Load correct Player Create XP Data

### LINKED ISSUE #17822: wrong experience
1 level should be 400, error 12000
starting with level 2, a strip of smooth experience level 1 = 400
wowscrnshot_081916_204139
wowscrnshot_081916_204123
7.0.3
https://github.com/TrinityCore/TrinityCore/commit/650ab693a8366847d7db8bd8443b3b4aaf99ff85

### PR DESCRIPTION
**Pull request by funjoker**
**Target branch(es):** 6.x

**Issues addressed:** Closes #17822 

**Tests performed:** (Does it build, tested in-game, etc)

**Known issues and TODO list:**
- Check other GT relevant stuff (Mitigation is working correct)

### CODE REVIEW

**Comment by funjoker** [comment]
Not sure if it was a unqiue failure or if it's a GT loading problem.

If it's a GT loading problem this should not be merged.

File: src/server/game/Globals/ObjectMgr.cpp [hunk 1]
```diff
@@ -3559,14 +3559,14 @@ void ObjectMgr::LoadPlayerInfo()
     {
         uint32 oldMSTime = getMSTime();
 
-        _playerXPperLevel.resize(sXpGameTable.GetTableRowCount() + 1, 0);
+        _playerXPperLevel.resize(sXpGameTable.GetTableRowCount() - 1, 0);
```
Comment (Shauren) [inline src/server/game/Globals/ObjectMgr.cpp]: this change is wrong, you will access `_playerXPperLevel[sXpGameTable.GetTableRowCount()]`

Comment (funjoker) [inline src/server/game/Globals/ObjectMgr.cpp]: Tested it. All Level xp were loaded.
The Table has 124 rows.

Row 0 is Header stuff.
So 123 rows with data.

Getting the row count and removing the Header will still leave 123 rows. Starting loading from Level 1 Not 0 (which doesnt exist) will cause full loading. 

But i can undo this Part

Comment (Shauren) [inline src/server/game/Globals/ObjectMgr.cpp]: sure, if the vector has allocated space for more rows. but you are still going out of bounds

**Pull request merged**